client/state/timezones/schema.js

```diff
@@ -0,0 +1,33 @@
+
+export const timezonesSchema = {
+	type: 'object',
+	additionalProperties: false,
```

Comment:
Generally we only need this when using a patternProperty. It'd allow anything outside of the regex otherwise.